nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/PutAzureEventHub.java

```diff
@@ -86,6 +86,13 @@
             .required(true)
             .build();
 
+    static final PropertyDescriptor PARTITION_KEY = new PropertyDescriptor.Builder()
+            .name("Partition ID for Event")
```

Comment:
Let's make the name consistent, this should be Partition KEY for Event, shouldn't it?